fixes web console introduction and addresses linking issues (#11609)
apache/druid: docs/operations/high-availability.md

We recommend either installing ZooKeeper on its own hardware, or running 3 or 5 Master servers (where overlords or coordinators are running)
and configuring ZooKeeper on them appropriately. See the [ZooKeeper admin guide](https://zookeeper.apache.org/doc/current/zookeeperAdmin) for more details.
- For highly-available metadata storage, we recommend MySQL or PostgreSQL with replication and failover enabled.

> See the "MySQL HA/Scalability Guide" and [PostgreSQL's High Availability, Load Balancing, and Replication](https://www.postgresql.org/docs/current/high-availability.html) for MySQL and PostgreSQL, respectively.

Reviewer: since the first link is missing, maybe we should rephrase to not have the title of the linked article ("MySQL HA/Scalability Guide")

Reviewer 2: I think for MySQL we can link to https://www.mysql.com/products/enterprise/high_availability.html.

> See the "MySQL HA/Scalability Guide" and [PostgreSQL's High Availability, Load Balancing, and Replication](https://www.postgresql.org/docs/current/high-availability.html) for MySQL and PostgreSQL, respectively.

Reviewer 2: ```suggestion
See [MySQL Enterprise High Availability](https://www.mysql.com/products/enterprise/high_availability.html) and [PostgreSQL's High Availability, Load Balancing, and Replication](https://www.postgresql.org/docs/current/high-availability.html) for more information.
```
